### ISSUE TITLE
Help for train_lora-scale.py file

### ISSUE THREAD

**Issue by W8Yi** [rohitgandikota/sliders#82]
for file `train_lora-scale.py`, could you explain what is `low_latents` and `high_latents`? and I am confused why you didn't use them. Also, there is `positive_latents`, `neutral_latents`, `unconditional_latents` coming from nowhere, is this a bug/mistake?  Thank you for help and I really like your work. 

![Screenshot 2024-03-13 205331](https://github.com/rohitgandikota/sliders/assets/95598753/40cf1bf4-9844-4605-af72-af6e3b0594c4)

**Comment by rohitgandikota** [comment]
You are right! Thanks for catching it! The positive, negative and unconditional are a bug. They are in verbose print statements. We will try to fix this bug.

The high latent and low latent are the paired noise prediction for the image pairs you create. For example in the eyebrow dataset we released, high latent is for wavy eyebrow prediction and low latent is the normal eyebrow prediction. Then we create a loss function as described in our paper.

**Comment by W8Yi** [comment]
Isn't that should be target_latents_high and target_latents_low, which are used in the loss?

**Comment by zhuyinna** [comment]
> You are right! Thanks for catching it! The positive, negative and unconditional are a bug. They are in verbose print statements. We will try to fix this bug.
> 
> The high latent and low latent are the paired noise prediction for the image pairs you create. For example in the eyebrow dataset we released, high latent is for wavy eyebrow prediction and low latent is the normal eyebrow prediction. Then we create a loss function as described in our paper.

I have same question, too. Could you please solve this issue

**Comment by rohitgandikota** [comment]
This is fixed in the latest releases. Thanks everyone for catching this and raising the issue
